internal/pkg/service/stream/storage/level/local/encoding/encoder/csv/fastcsv/writer.go

```diff
@@ -17,28 +17,35 @@ func newWriter(out io.Writer) *writer {
 	return &writer{out: out}
 }
 
-func (w *writer) WriteRow(cols *[]any) error {
+func (w *writer) WriteRow(cols *[]any) (int, error) {
+	// Bytes written
+	n := 0
+
 	// Reset re-used buffer
 	w.row.Reset()
 
 	// Write each column
-	for n, col := range *cols {
+	for i, col := range *cols {
```

Comment:
```suggestion
	for index, col := range *cols {
```
It's 2 numbers there and it would look better.